docs/howto/jsonapi/custom_type_value.md

```diff
@@ -0,0 +1,25 @@
+# How to customize the type of a JSON API document
+
+To change the type of a resource, just use the `type` class method in your serializer definition.
+
+Note that by default, type is pluralized.
```

Comment:
This is ambiguous. It seems to imply that the `type`, when specified through the `type` method will be subject to inflection.